### DISCUSSION TITLE
window11 title  no show

### DISCUSSION THREAD

**Discussion by yst-one** [flet-dev/flet#3489] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

11

### Code

import flet as ft

def main(page: ft.Page):
    page.title = "my blibli app"

    #page.add(ft.Text("Hello, World!"))
ft.app(main)
Do not comment   #page.add(ft.Text("Hello, World!"))  normal

### To reproduce

111

### Expected behavior

111

### Screenshots

![image](https://github.com/flet-dev/flet/assets/134467913/ce47b1d9-f64b-4b58-b84e-3f2dba5f4449)

### Operating System

Windows

### Operating system details

win11

### Flet version

22.1

### Regression

I'm not sure / I don't know

### Suggestions

s

### Additional details

s

**Comment by azhago** [comment]
Hi,
you need to update the page to change the title
```py
def main(page:ft.Page):
    page.title = "my blibli app"
    page.update()
    
if __name__ == "__main__":
    ft.app(target=main)
```
will work

![image](https://github.com/flet-dev/flet/assets/66644024/5a35b365-a2d0-4b09-94e0-d6f98a93c3e7)
